src/renderer/components/FileUploadStages/PickConfiguration/Stage2.tsx

```diff
@@ -115,28 +85,8 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 
-// Styled Material UI Table Cell component
-const TableCell = withStyles({
-    root: {
-        borderBottom: "none"
-    }
-})(MuiTableCell);
-
```

Comment:
blank lines?

Reply:
That piece of code does not exist anymore in this pull request, thats wy there is a '-' sign in front of the line 🙂